### DISCUSSION TITLE
Create a new "survivor" achievement for authoring to merge a PR that has 100+ review comments

### DISCUSSION THREAD

**Discussion by westonpace** [community/community#20896] [Other Feature Feedback, Questions, & Ideas]
The icon could be a zombie or something like that.

**Comment by devkennyy** [comment]
100 review comments haha, I don't really see as an achievement.

**Reply by JJTech0130** [reply]
But if it actually gets merged, after all that...

**Reply by ssbarnea** [reply]
Indeed that is an awesome stackable achievement.

**Comment by HD111995** [comment]
https://github.com/Schweinepriester/github-profile-achievements
